igbinary/igbinary.php

```diff
@@ -31,7 +31,7 @@ function igbinary_serialize($value) {}
  * In case the passed string is not unserializeable, NULL is returned and E_WARNING is issued.
  *
  * @param string $str The serialized string.
- * @return mixed The converted value is returned, and can be a boolean, integer, float, string, array or object.
+ * @return mixed|false The converted value is returned, and can be a boolean, integer, float, string, array, object or <b>false</b> by empty string input.
```

Comment:
`Mixed` already includes `false` so `false` here can be removed from typehint

Reply:
We already have many stubs that use "mixed|false" and I think it can help some developers to see that there is a kind of error handling that are needed to check, if you will properly use this. But if you think that it's not helpful, should we revert all "x|mixed|x" PHPDocs or only this?

Reply:
Fair enough. Let's leave it.